test/unit/model/v2/test_platforms_section.py

```diff
@@ -190,6 +192,7 @@ def _model_platforms_docker_errors_section_data():
             'dns_servers': [
                 int(),
             ],
+            "etc_hosts": int(),
```

Comment:
should this be str() instead of int() ?

Reply:
This is a test intended to trigger the error and compare the error message. Which, in this case, int() will trigger the error.